apps/fluent-tester/src/FluentTester/TestComponents/ActivityIndicator/ActivityIndicatorTest.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { ActivityIndicator } from '@fluentui-react-native/experimental-activity-indicator';
+import { Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { ACTIVITYINDICATOR_TESTPAGE } from './consts';
+
+const activityIndicatorTest: React.FunctionComponent<{}> = () => {
+  /** Bug: Customize doesn't do anything
+   * Tried having tokens not be props, but didn't work
+   * Not sure how to test/check where the tokens are passed through other than just looking at the final render
+   */
+  const CustomizedActivityIndicator = ActivityIndicator.customize({
+    activityIndicatorColor: '#458792',
+  });
+  return (
+    <Stack style={stackStyle}>
+      <Text>Extra Small</Text>
+      <ActivityIndicator size="xSmall" />
+      <Text>Small</Text>
+      <ActivityIndicator size="small" />
+      <Text>Medium</Text>
+      <ActivityIndicator size="medium" />
+      <Text>Large</Text>
+      <ActivityIndicator size="large" />
+      <Text>Extra Large</Text>
+      <ActivityIndicator size="xLarge" />
+
+      <Text>Size and Line Thickness props of different sizes</Text>
+      <ActivityIndicator size="xLarge" lineThickness="large" />
+      <Text>Color props</Text>
+      <ActivityIndicator activityIndicatorColor="#921571" />
+      <Text>Animating: False, hidesWhenStopped: False</Text>
+      <ActivityIndicator animating={false} hidesWhenStopped={false} />
```

Comment:
Could we have a section where we hook up the Boolean props to a switch? Similar to what I do [here](https://github.com/microsoft/fluentui-react-native/pull/682) (which I just copied elsewhere from FURN

Reply:
I tried adding switches but the animation does not behave as expected. I think it's because of the combination of the `useState` and `useEffect` with the animation it's somehow not rerendering correctly. Even though it should since my condition is inside of the hook.

https://user-images.githubusercontent.com/30802320/125356254-f4831280-e333-11eb-89d4-602ca1a01736.mov

Reply:
The weird jerking is because of native driver was set to true. If I set native driver to false, it stops jerking, but it never stops even when Animating is false. This is because there isn't a "stopAnimating" function, so when it re-renders, the Animated.loop isn't run but it's still animating from when Animating was true.